installers/rpm-code/codes/mantid/rsmantid.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+set -euo pipefail
+exec ${MANTID_PYTHON_SCRIPT_INSTALL_LOCATION} --classic
```

Comment:
Always quote unless it's necessary not to. Everything is an example. Use POSIT or ASSUME if you are not quoting something.

Reply:
8c3e1a0